config/custom.cfg

```diff
@@ -0,0 +1,17 @@
+[buildout]
+auto-checkout = Euphorie
+
+[settings]
+# main-package = your.euphorie.flavor
+saconfig-uri = sqlite:///app/var/quoira.sql
+
+[instance]
+# remove the zcml declaration if you configure a main-package
```

Comment:
base.cfg defines `Euphorie` as the default main-package. 
https://github.com/euphorie/euphorie-project/blob/6fdc6387772e66eee42a63e9f751df791bef71a7/config/base.cfg#L17

But it then proceeds to load the main package zcml
https://github.com/euphorie/euphorie-project/blob/6fdc6387772e66eee42a63e9f751df791bef71a7/config/base.cfg#L53

which errors out because Euphorie has no top level zcml